src/commands/image/imageSource/buildImageInAzure/TarFileStep.ts

```diff
@@ -16,7 +16,7 @@ export class TarFileStep extends AzureWizardExecuteStep<BuildImageInAzureImageSo
     public async execute(context: BuildImageInAzureImageSourceContext): Promise<void> {
         const id: number = Math.floor(Math.random() * Math.pow(10, idPrecision));
         const archive = `sourceArchive${id}.tar.gz`;
-        context.tarFilePath = Utils.joinPath(URI.parse(os.tmpdir()), archive).fsPath;
+        context.tarFilePath = path.join(os.tmpdir(), archive);
```

Comment:
The old carryover path logic sometimes wouldn't work in the pipelines, this fixes that.